sha512_mb/sha512_mb_x2_avx.asm

```diff
@@ -337,10 +337,11 @@ Lrounds_16_xx:
 	vmovdqa	[STATE+7*SHA512_DIGEST_ROW_SIZE],h
 
 	; update input pointers
-	add	inp0, IDX
-	mov	[STATE + _data_ptr_sha512 + 0*PTR_SZ], inp0
-	add	inp1, IDX
-	mov [STATE + _data_ptr_sha512 + 1*PTR_SZ], inp1
+	vmovq	xmm0, IDX
```

Comment:
After giving an extra thought, I don't think making this change is worth it, for two reasons:
1 - It's replacing 4 instructions with 5 instructions
2 - We are removing AVX code in the next few months (already mentioned it).
So could you drop the changes in the AVX implementations and leave just the other ones?
Many thanks for the work!

Reply:
Sure, I've just reverted the _avx functions to the old version.